src/Elliptic/Actions/InitializeSystem.hpp

```diff
@@ -56,9 +57,11 @@ struct InitializeSystem {
     using system = typename Metavariables::system;
     using fields_tag = typename system::fields_tag;
     using linear_operator_applied_to_fields_tag =
-        db::add_tag_prefix<LinearSolver::Tags::OperatorAppliedTo, fields_tag>;
+        db::variables_tag_with_tags_prefixed_by<
```

Comment:
You can leave these as-is, just using `db::add_tag_prefix` (same applies to many occurrences below)

(If your reasoning for this change is that you want to assert the tag is actually a variables tag: I think it's not the responsibility of the prefix utility metafunction to assert that. The class working with the tag, in this case `InitializeSystem`, should assert that what it receives is a variables tag if it expects one.)